### PR TITLE
ActivatableQuerySet to only trigger activation changed signal for models

### PR DESCRIPTION
**Pull request by thinkt4nk**
whose active state actually changed, rather than triggering for all
models in the queryset.

### CODE REVIEW

**Comment by thinkt4nk** [comment]
@wesokes @jaredlewis if you guys wouldn't mind giving this a once-over, I'd appreciate it

**Comment by wesokes** [comment]
i've looked over it and it looks good to me. we can let it run for a while on staging and mess around to make sure things are ok.

File: activatable_model/__init__.py [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # flake8: noqa
 from .models import BaseActivatableModel, ActivatableManager, ActivatableQuerySet
-from .signals import model_activations_changed
+from .signals import *
```
Comment (jaredlewis) [inline activatable_model/__init__.py]: Is there any way to avoid using * here?

**Comment by thinkt4nk** [comment]
good idea @wesokes

File: activatable_model/__init__.py (hunk 1 above)
Comment (thinkt4nk) [inline activatable_model/__init__.py]: oh, definitely

**Comment by jaredlewis** [comment]
This looks awesome, makes a lot of sense to split it up that way

**Comment by jaredlewis** [comment]
image

**Pull request merged**